lib/v8debugapi.js

```diff
@@ -198,7 +199,7 @@ module.exports.create = function(logger_, config_, fileStats_) {
             });
           });
 
-      listeners[num] = listener;
+      listeners[num] = { enabled: true, listener: listener };
```

Comment:
I think since this is only done twice in this file I may leave it as is. However, I'll keep an eye out for opportunities to class-ify things (especially if we can get some type-checking in place 😄) .